### ISSUE TITLE
No favicon

### ISSUE THREAD

**Issue by jbuller** [ministryofjustice/record-a-goose-sighting#8]
A trainee flagged this. It makes it hard to find the site amongst other tabs.
This is a usability issue rather than accessibility so I feel should be fixed. 

Ideas:  https://www.google.com/search?q=goose+icon&tbm=isch

**Comment by BNewing** [comment]
Thank you for raising this! Happy to add one in. Do you have any specific examples that don't need attribution? There are lots on things like the Noun Project, and they require attribution, but I'm not sure where you'd put that if it's a favicon and not directly embedded in the page.

**Comment by jbuller** [comment]
You can use the Tools > Usage Rights option on the Google Search
https://www.google.com/search?q=goose%20icon&tbm=isch&tbs=sur%3Af&hl=en&ved=0CAQQpwVqFwoTCOjqkdfEyucCFQAAAAAdAAAAABAC&biw=1522&bih=750

See also https://www.onehourprofessor.com/what-does-labeled-for-non-commercial-reuse-mean/ and similar explainations

**Comment by BNewing** [comment]
Added one in :)
